package.json

```diff
@@ -2,7 +2,7 @@
   "name": "nsfwjs",
   "version": "0.0.3",
   "description": "Detect NSFW content client side",
-  "main": "dist/index.js",
+  "private": true,
```

Comment:
```
Finally, if you do not wish to grant others the right to use a private or unpublished package under any terms:

{ "license": "UNLICENSED" }
Consider also setting "private": true to prevent accidental publication.
```
Why would I set private to true?